src/index.html

```diff
@@ -5,10 +5,39 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+            src="images/logo.png"
+            class="logo_image"
+            alt="Moyo logo">
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/e7eaa7f1-a748-4b45-af41-02d72c18727a">

Reply:
Could you explain what is wrong with that part?
<img width="256" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/66942958/462ee1eb-368a-4d84-9a2a-2815798a303b">

Reply:
fixed